pom.xml

```diff
@@ -366,6 +374,9 @@
                         <goals>
                             <goal>jar</goal>
                         </goals>
+                        <configuration>
+                            <javadocExecutable>${java.home}/bin/javadoc</javadocExecutable>
```

Comment:
Any reason this needed to set specifically?

Reply:
Yea my intellij environment ran into the issue described and solved here https://intellij-support.jetbrains.com/hc/en-us/community/posts/206880855-MavenReportException-Error-while-creating-archive-Unable-to-find-javadoc-command-The-environment-variable-JAVA-HOME-is-not-correctly-set- so thats what allowed me to execute this in intellij. We can continue the convo here https://github.com/feast-dev/feast/pull/854